source/extensions/clusters/redis/redis_cluster.cc

```diff
@@ -331,14 +417,30 @@ void RedisCluster::RedisDiscoverySession::onResponse(
     return;
   }
 
-  auto slots = std::make_unique<std::vector<ClusterSlot>>();
-
+  auto cluster_slots = std::make_unique<std::vector<ClusterSlot>>();
+
+  // CLUSTER SLOTS represents nested array of redis instances, like this:
+  //
+  // 1) 1) (integer) 0                <-- start slot range
+  //    2) (integer) 5460             <-- end slot range
```

Comment:
What is the meaning of 5460 in this example?

Reply:
I'm not 100% sure, but looking at https://redis.io/topics/cluster-spec, it looks like the start-end defines a range between [0, 16384), that sets the upper limit for the cluster size.

Reply:
More info on this: 5460 is roughly equal to 16384/3, redis hashes keys and picks a server to store them on based on 16384 mod n where n is number of servers in the cluster